### PR TITLE
Migrate multiple storage class selection to odf

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Signed-off-by: Timothy Asir Jeyasingh <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@bipuladh Please review

File: cypress/tests/bucket-class-spec.ts [hunk 1]
```diff
@@ -11,8 +11,16 @@ import {
 } from '../views/bc';
 
 describe('Tests creation of Standard Bucket Class', () => {
```
Comment (TimothyAsirJeyasing) [inline cypress/tests/bucket-class-spec.ts:13]: This file was added when i run the lint fix and yarn format command.
Will it be ok or shall i remove this ?

Comment (bipuladh) [reply cypress/tests/bucket-class-spec.ts:13]: It is okay.

**Comment by TimothyAsirJeyasing** [comment]
Screenshot from 2022-06-17 13-45-04
Screenshot from 2022-06-17 13-45-36

**Pull request merged**